.circleci/config.yml

```diff
@@ -1507,18 +1509,32 @@ jobs:
       - CARGO_TARGET_DIR: /mnt/ramdisk/cargo
     steps:
       - checkout
+      - <<: *STEP_ATTACH_WORKSPACE
       - restore_cache:
           name: Restore Cargo Package Cache
           keys:
             - profiling-cargo-cache-<< parameters.triplet >>-{{ checksum "profiling/Cargo.toml" }}
       - run:
-          name: Test Profiler
+          name: cargo tests
           command: |
             set -u
             command -v switch-php && switch-php "${PHP_VERSION}"
             set -e
             cd profiling
             cargo test --release
+      - run:
+          name: phpt tests
+          command: |
+            set -u
+            command -v switch-php && switch-php "${PHP_VERSION}"
+            set -e
+            libdir=$PWD/datadog-profiling/<< parameters.triplet >>/lib/php/<< parameters.abi_no >>
+            cd profiling/tests
+            # don't anticipate there being more than one
+            run_tests_php=$(find $(php-config --prefix) -name run-tests.php)
+            cp -v "${run_tests_php}" .
+            export TEST_PHP_EXECUTABLE=$(which php)
+            php run-tests.php -d "extension=${libdir}/datadog-profiling.so" phpt
```

Comment:
We're generally running run-tests with a `--show-diff -g FAIL,XFAIL,BORK,WARN,LEAK,XLEAK,SKIP` incantation, maybe it makes sense to align this for profiler tests?